### ISSUE TITLE
Wait for real confirmation at the warning "DESTINATION WILL BE PERMANENTLY CHANGED!"

### ISSUE THREAD

**Issue by klemens-u** [theraser/blocksync#10]
Connected to https://github.com/theraser/blocksync/issues/9

> Add option to disable "DESTINATION WILL BE PERMANENTLY CHANGED!" #9 

If the above change would be implemented it would make to sense to wait for a real user input to make the command safer.
Example:

```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!                                          !!!
!!! DESTINATION WILL BE PERMANENTLY CHANGED! !!!
!!!         DO YOU WANT TO PROCEED? Y/n      !!!
!!!                                          !!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by devZer0** [comment]
yes, i think it's unfortunate that there is a warning printed and then ist starts automatically if you are not quick enough. 

but i can see the intention - you have a chance to interrupt if you accidentally select the last command from shell history for example.

anyhow, having timeout in a tool which does destructive operation is not something i did see in 20 years of unix, normally you would be interactively queried as you suggest or you would typically need to specify "-y" (= yes, i'm really sure, i know what i'm doing)
